### PR TITLE
SUBMARINE-286. Add documentation of E2E test framework

### PR DESCRIPTION
**Pull request by jasoonn**
### What is this PR for?
To complete the documentation of E2E test framework of this project.

### What type of PR is it?
Documentation

### What is the Jira issue?
https://issues.apache.org/jira/browse/SUBMARINE-286

### How should this be tested?
[CI Pass](https://travis-ci.org/apache/submarine/builds/608632568?utm_medium=notification&utm_source=github_status)

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

**Comment by jasoonn** [comment]
> @jasoonn @kevin85421 ,
> 
>     1. You need create a new JIRA for this RP.
> 
>     2. Need change PR title to `SUBMARINE-JIRAID. xxxxx`

I wiil work on it.

**Comment by jasoonn** [comment]
> @jasoonn @kevin85421 ,
> 
>     1. You need create a new JIRA for this RP.
> 
>     2. Need change PR title to `SUBMARINE-JIRAID. xxxxx`

I have finished these two things. Thank you!

**Comment by jasoonn** [comment]
> @jasoonn @kevin85421 ,
> 
>     1. You need create a new JIRA for this RP.
> 
>     2. Need change PR title to `SUBMARINE-JIRAID. xxxxx`

@liuxunorg 
I have finished these two things. Thank you!

File: docs/development/IntegrationTest.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+```
+BUILD SUCCESS
+```
+Otherwise, it will show.
+```
+BUILD FAILURE
+```
+
+## Add your own integration test
+1. Create new file ending with "IT" under "hadoop-submarine/submarine-test/e2e/src/test/java/org/apache/submarine/integration/".
```
Comment (liuxunorg) [inline docs/development/IntegrationTest.md]: Need change `hadoop-submarine/`  to `submarine/`

File: docs/development/IntegrationTest.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  See the License for the specific language governing permissions and      
+  limitations under the License. See accompanying LICENSE file.      
+-->
+
+# IntegrationTest
+
+## Run the existing tests.
+##### Move to the working directroy.
+```
+cd hadoop-submarine/submarine-test/e2e
```
Comment (liuxunorg) [inline docs/development/IntegrationTest.md]: Need change `hadoop-submarine` to `submarine`.

Comment (jasoonn) [reply docs/development/IntegrationTest.md]: > Need change `hadoop-submarine` to `submarine`.

@LinhaoZhu ,
I have changed above two things. Thank you.

**Comment by liuxunorg** [comment]
Will merge if no more comment.

**Pull request closed without merging**